package.json

```diff
@@ -65,9 +65,12 @@
     "ember-source": "~3.1.0",
     "ember-source-channel-url": "^1.0.1",
     "ember-try": "^0.2.23",
+    "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-ember": "^5.0.0",
     "eslint-plugin-node": "^6.0.1",
-    "loader.js": "^4.2.3"
+    "eslint-plugin-prettier": "2.6.0",
```

Comment:
This dependency's version is fixed because of this problem: https://github.com/ember-cli/ember-cli-eslint/issues/235